e2e/test/scenarios/dashboard/dashboard.cy.spec.js

```diff
@@ -693,7 +693,9 @@ function checkOptionsForFilter(filter) {
     .and("not.contain", "Dashboard filters");
 
   // Get rid of the open popover to be able to select another filter
-  cy.findByText("Pick one or more filters to update").click();
+  // Uses force: true because the popover is covering this text. This happens
+  // after we introduce the database prompt banner.
+  cy.findByText("Pick one or more filters to update").click({ force: true });
```

Comment:
Can we scroll to it somehow? e.g. scrollIntoView or similar

Reply:
This is the place that this test is clicking
![Screenshot 2023-06-08 at 4 36 09 PM](https://github.com/metabase/metabase/assets/1937582/e31e3dd2-0def-4e9f-bd14-852200ac0f2f)

Unfortunately, this is a full-height layout. So it's impossible to scroll anywhere on this page for this particular setup. The problem is when clicking the **Category** filter. It will look like the following screenshot which completely cover the text element we want to click.
![image](https://github.com/metabase/metabase/assets/1937582/d9123979-a485-4725-a5bc-a3bf84295bf4)